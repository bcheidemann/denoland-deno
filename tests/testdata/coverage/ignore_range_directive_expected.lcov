SF:[WILDCARD]ignore_range_directive.ts
FN:7,used
FNDA:1,used
FNF:1
FNH:1
BRF:0
BRH:0
DA:1,1
DA:2,1
DA:7,1
LH:3
LF:3
end_of_record
