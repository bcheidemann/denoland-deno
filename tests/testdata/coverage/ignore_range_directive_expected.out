cover [WILDCARD]/coverage/ignore_range_directive.ts ... 100.000% (3/3)
